nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -517,20 +541,70 @@ def streaming_post_process(self, rets, keep_all_outputs=True):
 
     @typecheck()
     def forward(
-        self, audio_signal, length, cache_last_channel=None, cache_last_time=None, cache_last_channel_len=None
+        self,
+        audio_signal,
+        length,
+        cache_last_channel=None,
+        cache_last_time=None,
+        cache_last_channel_len=None,
+        pre_encode_input=False,
```

Comment:
The name `pre_encode_input` is a bit ambiguous, which seems to suggest that it's used to directly inject the pre_encode features by skipping the pre_encode module. Maybe use something like `return_pre_encode`?